dist/react-formbuilder.js

```diff
@@ -1331,26 +1331,91 @@ module.exports = React.createClass({
 "use strict";
 
 
-var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
-
-var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };
-
 var React = __webpack_require__(0);
 var cleanProps = __webpack_require__(1);
 var MultiplicityField = __webpack_require__(5);
 
 module.exports = React.createClass({
   displayName: "exports",
+  getInitialState: function getInitialState() {
+    console.log(this.props);
+    return {
+      attachment: false
+    };
+  },
   render: function render() {
+    var _this = this;
```

Comment:
it's the file that you get when you `require('react-formbuilder')`, `.npmignore` skips pretty much everything else because it's irrelevant when you just want to use the library as dependency.